install.sh

```diff
@@ -2,7 +2,7 @@
 set -eu
 
 # code-server's automatic install script.
-# See https://github.com/cdr/code-server/blob/main/docs/install.md
+# See https://github.com/cdr/code-server/blob/main/docs/install
```

Comment:
Should this also go to coder.com? Without `.md` it seems to 404.

Reply:
Dang it. I did a serach for `.md` and must have accidentally done a search and replace. I'll fix.

Reply:
https://github.com/cdr/code-server/pull/3711